test/e2e/scheduler_test.go

```diff
@@ -159,3 +176,64 @@ func TestConfigMapCreation(t *testing.T) {
 		t.Fatalf("error waiting for config map to exist: %v\n", err)
 	}
 }
+
+func waitForOperatorToBeUpdated(opClient operatorclientv1.OperatorV1Interface, status string) error {
```

Comment:
this is actually waiting for it to be removed, right?  Rename please

Reply:
Done